lib/setopt.c

```diff
@@ -2324,7 +2324,7 @@ CURLcode Curl_vsetopt(struct Curl_easy *data, CURLoption option, va_list param)
         Curl_hsts_cleanup(&data->hsts);
         data->hsts = data->share->hsts;
       }
-#endif   /* CURL_DISABLE_HTTP */
+#endif   /* CURL_DISABLE_HSTS */
```

Comment:
Nice catch, when pushing I will probably just remove this comment altogether instead since the `ifndef` is vertically very close and leaving these comments out on short blocks matches the style of the file.